crates/autopilot/src/driver_model.rs

```diff
@@ -150,6 +153,36 @@ pub mod solve {
         pub submission_address: H160,
     }
 
+    #[derive(Clone, Debug, Serialize, Deserialize)]
+    #[serde(rename_all = "camelCase")]
+    pub enum FeePolicy {
+        /// If the order receives more than expected (positive deviation from
+        /// quoted amounts) pay the protocol a factor of the achieved
+        /// improvement. The fee is taken in `sell` token for `buy`
+        /// orders and in `buy` token for `sell` orders.
+        QuoteDeviation {
```

Comment:
nit: can we either call this `PositiveQuoteDeviation` or `PriceImprovement` ?

Reply:
renamed